docs/integrations/duckdb.md

```diff
@@ -18,15 +18,34 @@ Querying data in lakeFS from DuckDB is similar to querying data in S3 from DuckD
 
 ### Configure the lakeFS S3 Gateway endpoint
 
+If not loaded already, install and load the HTTPFS extension: 
+
 ```sql
+INSTALL httpfs;
 LOAD httpfs;
+```
+
+Then run the following to configure the connection. 
+
+```sql
 SET s3_region='us-east-1';
 SET s3_endpoint='lakefs.example.com';
 SET s3_access_key_id='AKIAIOSFODNN7EXAMPLE';
 SET s3_secret_access_key='wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY';
 SET s3_url_style='path';
 ```
 
+* `s3_endpoint` is the host (and port, if necessary) of your lakeFS server
+* `s3_access_key_id` and `s3_secret_access_key` are the access credentials for your lakeFS user
+* `s3_url_style` needs to be set to `path`
+* `s3_region` is the S3 region on which your bucket resides. If local storage, or not S3, then just set it to `us-east-1`. 
+
+If lakeFS is not running on an SSL-secured server (for example, you are just running it locally) then you need to also run
+
+```sql
+SET s3_use_ssl=false;
+```
```

Comment:
Suggestion:
Can we move this one to the "configure the connection" section as a comment just as example on what to set in case we are working with a secure connect (same as your comment).
It just more for the structure as we have section with all the settings and the following explanation.

Reply:
ah ok, thanks. updated per your suggestion.